src/app/api/users/[userId]/profile/route.ts

```
@@ -52,9 +84,9 @@ export async function GET(
             profileBackgroundColor: profile.profileBackgroundColor,
             profileBackgroundGradient: profile.profileBackgroundGradient,
             profileBackgroundImageFileId: profile.profileBackgroundImageFileId,
-            profileBackgroundUrl,
+            profileBackgroundUrl: profileBackgroundUrl,
```

ESLint

[error] Expected property shorthand.

(object-shorthand)